packages/worker/src/server-options.ts

```diff
@@ -1,13 +1,15 @@
 import os from 'os';
-import ms from 'ms';
-export { normalizeTlsConfig } from '@temporalio/common';
 import * as native from '@temporalio/core-bridge';
 import pkg from './pkg';
+import path from 'path';
 
 type TLSConfig = native.TLSConfig;
 
 export { TLSConfig };
 
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const sdkVersion = require(path.resolve(__dirname, '../package.json')).version;
```

Comment:
We already read package.json in `pkg`, we should probably just use that.
Shouldn't the SDK version also include the SDK name?